### ISSUE TITLE
Scribe using type Laravel hardcodes URL at generate time

### ISSUE THREAD

**Issue by ElvenSpellmaker** [knuckleswtf/scribe#431]
- [x] **I confirm that I have read and attempted the tips in the [Troubleshooting Guide](https://scribe.knuckles.wtf/laravel/troubleshooting).**

**What happened?**
1. I set my configuration to `type => 'laravel',`
2. Then I ran `php artisan scribe:generate` and it generates the blade template
3. But I saw the APP_URL is hardcoded into the blade template.

instead of using APP_URL at render time.

The problem we have is that we build a Docker image one and use it later and don't have the URLs  at the build time and so hardcoded URL from the `.env` isn't appropriate.

E.g. APP_URL is set to the correct URL on the container, however examples etc are all using the value from `.env` at build time:
![image](https://user-images.githubusercontent.com/2286713/153252167-d3fc3540-ffa4-4428-9d15-3979ce52fd11.png)

![image](https://user-images.githubusercontent.com/2286713/153251787-7a65ec08-65d2-420c-bc53-d3552a7e5429.png)

**My environment:**
 - PHP version (from `php -v`): N/A
 - Framework (Laravel/Lumen): Lumen 8.x
 - Scribe version (from `composer show knuckleswtf/scribe`): latest stable

**Comment by shalvah** [comment]
Did you check the config file or docs? https://scribe.knuckles.wtf/blog/2021/06/08/laravel-v3#multiple-urls

**Comment by ElvenSpellmaker** [comment]
@shalvah Yeah I checked the docs, didn't realise it'd be in a section about multiple URLs because I don't have multiple URLs.

Either way this doesn't fix my problem because it seems to put the URL into blade at _build time_ and not at _runtime)

**Comment by shalvah** [comment]
Not really, Scribe is fairly extensible, so you're hardly ever "stuck". Yes, it puts the URLs in at build time. We could change that for `Laravel` type, but that would be extra complexity and not everyone would want to use APP_URL. 

However, you can override the default templates and replace the URL part with your dynamic version. There's a section in the docs about customising.

**Comment by shalvah** [comment]
Oh, another option: you could use the `afterGenerating()` hook to do a search and replace.
